app/models/TraderProfile.scala

```diff
@@ -109,4 +109,17 @@ object TraderProfile {
       case Left(errors) => Left(errors)
     }
 
+  def validateNirmsNumber(
```

Comment:
Validates whether the answers are valid for particular inputs.

Reply:
why do we need this - we use buildNirms

Reply:
I will check if I can use buildNirms instead :)